f2/src/utils/submitReportToServer.js

```diff
@@ -0,0 +1,31 @@
+const flatten = require('flat')
```

Comment:
`submitReportToServer.js` is just a utility to allow us to poke at the server without having to go through the app